woocommerce/woocommerce-ios
# Missed Podfile.lock checksum change from WordPressAuthenticator pod version update.

### Description
Running `bundle install && bundle exec rake dependencies` from `trunk` introduces local changes the `PODFILE CHECKSUM` `Podfile.lock` file. 

I am guessing that this is because I didn't merge `trunk` back into my branch to [get these changes](https://github.com/woocommerce/woocommerce-ios/commit/9ca46e2906498a3adaa3d8569638f57c738deb32) before merging my PR https://github.com/woocommerce/woocommerce-ios/pull/7424. 

I am raising this PR to commit the `Podfile.lock` file changes into `trunk`.

### Testing instructions
- Run `bundle install && bundle exec rake dependencies`
- Run `git status`
- Ensure that the working tree is clean. (There are new local changes cocoapods related files.)

### Screenshots
NA

---
